RecoTracker/TkSeedGenerator/python/trackerClusterCheck_cfi.py

```diff
@@ -26,3 +26,12 @@
                MaxNumberOfPixelClusters = 150000,
                MaxNumberOfCosmicClusters = 500000
                )
+
+from Configuration.ProcessModifiers.egamma_lowPt_exclusive_cff import egamma_lowPt_exclusive
+egamma_lowPt_exclusive.toModify(trackerClusterCheck,
+               doClusterCheck=True,
+               cut = "strip < 1000 && pixel < 300 && (strip < 50000 + 10*pixel) && (pixel < 5000 + 0.1*strip)",
```

Comment:
Aren't the third and fourth conditions in AND already included in the first two?

Reply:
I agree.
@rchudasa please remove unnecessary parts of the cut

Reply:
@perrotta , @slava77 
Thanks, removed it. :)